# HG changeset patch
# User jwe
# Date 848427198 0
#      Tue Nov 19 18:13:18 1996 +0000
# Node ID d21eb2d6e135d80ee12246264fe16a0882b755d2
# Parent  64f403ef483d95558e9e803204f4cde4f9d6e364
[project @ 1996-11-19 18:13:18 by jwe]
pt-exp.h

diff --git a/src/pt-exp.cc b/src/pt-exp.cc
--- a/src/pt-exp.cc
+++ b/src/pt-exp.cc
@@ -783,25 +783,16 @@ tree_simple_assignment_expression::eval_
 void
 tree_simple_assignment_expression::accept (tree_walker& tw)
 {
   tw.visit_simple_assignment_expression (*this);
 }
 
 // Colon expressions.
 
-bool
-tree_colon_expression::is_range_constant (void) const
-{
-  bool tmp = (op_base && op_base->is_constant ()
-	      && op_limit && op_limit->is_constant ());
-
-  return op_increment ? (tmp && op_increment->is_constant ()) : tmp;
-}
-
 tree_colon_expression *
 tree_colon_expression::chain (tree_expression *t)
 {
   tree_colon_expression *retval = 0;
   if (! op_base || op_increment)
     ::error ("invalid colon expression");
   else
     {
